demos/python_demos/object_detection_demo_faceboxes/detector.py

```diff
@@ -0,0 +1,200 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+import itertools
+import math
+import numpy as np
+import os
+import cv2
+
+from openvino.inference_engine import IENetwork
+
+class Detector(object):
+    def __init__(self, ie, model_path, device='CPU'):
+        model = IENetwork(model=model_path, weights=os.path.splitext(model_path)[0] + '.bin')
```

Comment:
IENetwork is deprecated, please follow to updated api and use IECore().read_network()

Reply:
Done